test/uk/gov/hmrc/entrydeclarationdecision/services/ProcessDecisionServiceSpec.scala

```diff
@@ -293,6 +341,8 @@ class ProcessDecisionServiceSpec
         MockOutcomeConnector.send(validOutcome(MessageType.IE316)) returns Future.successful(Left(someErrorCode))
 
         service.processDecision(decision).futureValue shouldBe Left(someErrorCode)
+
+        Thread.sleep(100) // check that MockStoreConnector.setShortTtl(submissionId) isn't called
```

Comment:
Why only in here and not in other cases where fail to save? Should this be a separate single test with a title that indicates that 'setShortTtl is not called when saving the outcome fails' or something?